Source/WebCore/platform/graphics/egl/GLContext.h

```diff
@@ -21,6 +21,7 @@
 
 #if USE(EGL)
 #include "GLContextWrapper.h"
+#include "IntRect.h"
```

Comment:
I think this could be forward declared instead.

Reply:
done